server/services/crmDisplayService.ts

```diff
@@ -123,7 +123,12 @@ export default class CrmDisplayService {
   }
 
   private getApiFieldValue<T extends CrmResponse>(crmResponse: T, apiFieldName: string): string {
-    return _.get(crmResponse, `formDetails.${apiFieldName}`) || _.get(crmResponse, apiFieldName) || ''
+    return (
+      _.get(crmResponse, `formDetails.${apiFieldName}`) ||
+      _.get(crmResponse, apiFieldName) ||
+      _.get(crmResponse, `mergedScheduleCostsData.${apiFieldName}`) ||
```

Comment:
`getApiFieldValue` method should be agnostic for all crm forms. `mergedScheduleCostsData` is only crm7 related, so I don't think this is the appropriate place for this.